test/remote/gateways/remote_paymentez_test.rb

```diff
@@ -34,6 +34,18 @@ def test_successful_purchase_with_more_options
     assert_success response
   end
 
+  def test_no_phone
```

Comment:
Let's make this more descriptive, something like `test_successful_purchase_without_phone_option`